### ISSUE TITLE
color-function-notation: legacy not working as expected

### ISSUE THREAD

**Issue by federico-bellucci** [stylelint/stylelint#5691]
### What steps are needed to reproduce the bug?

Have a CSS file with the rule: 
`color: rgba(0, 0, 0, 0.2);`

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "color-function-notation": "legacy",
  }
}
```

### How did you run Stylelint?

node package `stylelint 'src/**/*.css'`

### Which version of Stylelint are you using?

`14.0.1`

### What did you expect to happen?

The rule: 
`color: rgba(0, 0, 0, 0.2);`
should be accepted. 

### What actually happened?

The following problem was reported: 
Expected "0.2" to be "20%" (alpha-value-notation)

### Does the bug relate to non-standard syntax?

No

**Comment by federico-bellucci** [comment]
I now see that the the rule `'alpha-value-notation': 'percentage'` inherited from `stylelint-config-standard` is taking precedence over `"color-function-notation": "legacy"`
